src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -5,46 +5,42 @@ summary:
 
 # Carousel Pattern
 
-The Carousel Pattern
-enables you to display multiple items in a horizontal slide. Elements inside
-the placeholder will be split into different items. For dynamic content, use a
-List directly inside the placeholder. The Carousel block is ideal to create
-horizontal slides in smaller screens.
-
-Here's the preview in Service Studio:
+You can use the Carousel UI Pattern to display multiple items in a horizontal slide. This pattern is ideal for creating horizontal slides in smaller screens. You can also use this pattern for dynamic content, by placing a List directly inside the placeholder. 
 
 ![](images/Carousel_studio_preview.png)
 
-## How to Use the Carousel Pattern
+## How to use the Carousel UI Pattern
```

Comment:
This section is very rich, but the organization isn't great. Would a small table of contents here help, saying all the different things we'll cover? It's not easy to understand what's a standalone example vs steps to create the onboarding Carousel or even something else.